### PR TITLE
Apache: refactor integration

### PR DESCRIPTION
**Pull request by mtojek**
## What does this PR do?

This PR fixes a couple of action items in https://github.com/elastic/integrations/issues/370#issuecomment-733691243 .

Changes:
- Update Docker image to latest version
- Add information about supported OSes
- Update period for the status dataset
- Duplicate field defintions in apache.status to cover new ECS fields (uptime, ~~hostname~~). _The hostname is already defined in `agent.yml`_

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [x] I have verified that all datasets collect metrics or logs.

## Related issues

- https://github.com/elastic/integrations/issues/370

### CODE REVIEW

**Comment by mtojek** [comment]
jenkins run the tests please

File: packages/apache/_dev/build/docs/README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ logs created by the Apache server.
 
 ## Compatibility
 
-The Apache datasets were tested with Apache 2.4.12 and 2.4.20 and are expected to work with
-all versions >= 2.2.31 and >= 2.4.16.
+The Apache datasets were tested with Apache 2.4.12 and 2.4.46 and are expected to work with
+all versions >= 2.2.31 and >= 2.4.16 (independent from operating system).
```
Comment (ycombinator) [inline packages/apache/_dev/build/docs/README.md:9]: Some day it would be nice to auto-generate this section, or at least the versions in it from the https://github.com/elastic/integrations/blob/master/packages/apache/_dev/deploy/variants.yml file. We could probably take advantage of the README template for this. Not for this PR, just wanted to express the idea for now.

File: packages/apache/_dev/deploy/docker/httpd.conf [hunk 2]
```diff
@@ -149,6 +149,7 @@ LoadModule dir_module modules/mod_dir.so
 #LoadModule userdir_module modules/mod_userdir.so
 LoadModule alias_module modules/mod_alias.so
 #LoadModule rewrite_module modules/mod_rewrite.so
+LoadModule mpm_event_module modules/mod_mpm_event.so
```
Comment (ycombinator) [inline packages/apache/_dev/deploy/docker/httpd.conf:152]: Just curious: why was this change needed?

**Review by ycombinator — APPROVED** [review]
Left a comment with a long-term idea (not for this PR) and a question about a specific change. But otherwise LGTM.

**Comment by mtojek** [comment]
jenkins run the tests please

File: packages/apache/_dev/deploy/docker/httpd.conf (hunk 2 above)
Comment (mtojek) [reply packages/apache/_dev/deploy/docker/httpd.conf:152]: It was a quick fix. It complained about: `Configuration error: No MPM loaded`.

**Comment by mtojek** [comment]
jenkins run the tests please

**Comment by mtojek** [comment]
jenkins run the tests please
